src/main/java/it/pagopa/interop/probing/eservice/operations/model/view/EserviceView.java

```diff
@@ -38,30 +42,24 @@ public class EserviceView implements Serializable {
   @Column(name = "id", updatable = false)
   private Long id;
 
-  @NotBlank
   @Size(max = 255)
   @Column(name = "eservice_name")
   private String eserviceName;
 
-  @NotNull
```

Comment:
I didn't mean to remove these constraints, my comment was about the default values 😅 
We can leave it if it is useful

Reply:
Being a view and being read-only, there is no need to revalidate everything, so we agreed to remove this annotation from the view